src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -29,6 +29,130 @@ def setUp(self):
+             reg_conditional_device_data['installationParam']['latitude']
+            self.assertEqual(latitude, cbsd_record[0]['installationParam']['latitude'])
+            
+            longitude = device['installationParam']['longitude'] if ['longitude'] in device['installationParam'] else\
+             reg_conditional_device_data['installationParam']['longitude']           
+            self.assertEqual(longitude, cbsd_record[0]['installationParam']['longitude'])
+            
+            height = device['installationParam']['height'] if ['height'] in device['installationParam'] else\
+             reg_conditional_device_data['installationParam']['height']           
+            self.assertEqual(height, cbsd_record[0]['installationParam']['height'])
+            
+            height_type = device['installationParam']['heightType'] if ['heightType'] in device['installationParam'] else\
+             reg_conditional_device_data['installationParam']['heightType']           
+            self.assertEqual(height_type, cbsd_record[0]['installationParam']['heightType'])
+                    
+            antenna_gain = device['installationParam']['antennaGain'] if ['antennaGain'] in device['installationParam'] else\
+             reg_conditional_device_data['installationParam']['heightType']           
+            self.assertEqual(antenna_gain, cbsd_record[0]['installationParam']['antennaGain'])
+             # parameters should exist in record if exist in device,\
+             # if exist only in record, its value should be equal to a default valu           
+            self.assertEqualToIfExistDeviceParamOrPreloadedCondtional('indoorDeployment', \
+                device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'])
+            
+            
+            self.assertEqualToIfExistDeviceParamOrPreloadedCondtional('antennaAzimuth', \
+                device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'])
+            
+            self.assertEqualToIfExistDeviceParamOrPreloadedCondtional('antennaDowntilt', \
+               device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'])
+            
+            self.assertEqualToIfExistDeviceParamOrPreloadedCondtional('antennaBeamwidth', \
+               device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'])
+            
+            # if callSign exist, it should have the same value as registered
+            if 'callSign' in cbsd_record[0]:
+                self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])
+            max_eirp_by_MHz = 37;           
+            # if eirpCapability in the record, it should be the same as device or in accepted limits
```

Comment:
I don't think so, but this will probably depend on the outcome of our other discussion re: default values.

Reply:
I didn't see any conclusion to the discussion on default values so I'm commenting to keep this comment "alive" (I'm worried I'll lose track of it).

Reply:
Can you start a discussion (WG3?) about this issue? I'm pretty sure we're not supposed to fill in the default values.

Reply:
I think I need to update this comment to be:
 # if eirpCapability in the record, it should be the same as device or less than Fcc maxEirp value